tests/checkpoint/test_checkpoint.py

```diff
@@ -919,7 +1075,7 @@ def test_newstyle_checkpoint_config_substitution_simple(
                 }
             },
         ],
-        run_name_template="%Y-%M-foo-bar-template-$VAR",
+        run_name_template="%Y-%M-foo-bar-template-test",
```

Comment:
Why are we commenting out the line with `$VAR`?  I thought that this PR would make it work?..

Reply:
Oh. Not commenting out or removing anything here. I agree it's a bit confusing since there's tons of stuff in those tests.

The above change is defined within the expected output, hence all the replacements since var substitution works now.